src/Core/Services/Implementations/SsoConfigService.cs

```diff
@@ -42,15 +46,23 @@ public async Task SaveAsync(SsoConfig config)
             }
 
             var oldConfig = await _ssoConfigRepository.GetByOrganizationIdAsync(config.OrganizationId);
-            if (oldConfig?.GetData()?.UseKeyConnector == true && !useKeyConnector)
+            var disabledKeyConnector = oldConfig?.GetData()?.UseKeyConnector == true && !useKeyConnector;
+            if (disabledKeyConnector && await UserHasKeyConnectorEnabledAsync(config.OrganizationId))
             {
-                throw new BadRequestException("KeyConnector cannot be disabled at this moment.");
+                throw new BadRequestException("Key Connector cannot be disabled at this moment.");
             }
 
             await LogEventsAsync(config, oldConfig);
             await _ssoConfigRepository.UpsertAsync(config);
         }
 
+        private async Task<bool> UserHasKeyConnectorEnabledAsync(Guid organizationId)
```

Comment:
```suggestion
        private async Task<bool> HasAOrgUserEnabledKeyConnectorAsync(Guid organizationId)
```

Reply:
@djsmith85 What do you think of `AnyOrgUserHasKeyConnectorEnabledAsync` I think including `Any` makes it even more explicit.

Reply:
@Hinton That works for me. I tend to prefix Has/Is/Does with bools, hence the suggestion

Reply:
I think as long as it's formulated as a question with a yes/no response it's fine.